internal/cli/apis.go

```diff
@@ -113,9 +114,16 @@ auth0 apis show <id>
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if len(args) == 0 {
-				if err := apiID.Ask(cmd, &inputs.ID); err != nil {
+				err := apiID.Pick(cmd, &inputs.ID, cli.apiPickerOptions)
+				if err != nil {
 					return err
 				}
+
+				if inputs.ID == "" {
```

Comment:
Why the empty check if the options function already returns an error if there are no apps?